testsuite/pytests/test_stdp_synapse.py

```diff
@@ -104,7 +142,13 @@ def do_the_nest_simulation(self):
         """
         nest.set_verbosity("M_WARNING")
         nest.ResetKernel()
-        nest.SetKernelStatus({"resolution": self.resolution})
+        nest.SetKernelStatus(
+            {
+                "resolution": self.resolution,
+                "min_delay": min(self.min_delay, self.dendritic_delay),
```

Comment:
Why do you need to set min and max_delay explicitly?

Reply:
Because STDP (in NEST) strongly depends on when pre-synaptic spikes are communicated. And in theory things might become funky when spikes are communicated in different intervals than the delay of the specific connection. And things get especially funky when axonal delays are involved, so this is mainly a preparation for the future.

Reply:
OK, cool! I just thought there could be an option like "None" in there, which would allow NEST to compute the min and max-delay values internally.

Reply:
Not sure if this is necessary, as there will be other tests ensuring that the min_delay is always set correctly. For this test it is fine to always set the min and max delay explicitly.